example/config.yaml

```diff
@@ -18,12 +18,14 @@ settings:
     # configured Juniper equipment reports data over the JTI stream.
     interval: 1s
   write:
-    enabled: false
+    disable: true
```

Comment:
i'm not opposed to this, its just kinda weird to have the params inverted from almost all the other charts that implement enabled. 

Any reason why we're deviating? just preference?

Reply:
oh I know. it was a configuration scheme added early on to the SDK and has only made me sadder as time has moved on. 

I could deprecate  and add support for `enabled` to make everything consistent.

(note: the  `enabled: false` config from before was a mis-configuration, hence the change - the SDK doesn't support "enabled" as a key there)